primitives/src/v1/mod.rs

```diff
@@ -1216,6 +1220,17 @@ pub struct DisputeState<N = BlockNumber> {
 	pub concluded_at: Option<N>,
 }
 
+// FIXME
+#[cfg(feature = "std")]
+impl<N> MallocSizeOf for DisputeState<N> {
+	fn size_of(&self, _ops: &mut MallocSizeOfOps) -> usize {
+		0
+	}
+	fn constant_size() -> Option<usize> {
+		Some(0)
+	}
```

Comment:
this probably should probably not implemented, because it's not constant

Reply:
Git